src/backend/commands/createas.c

```diff
@@ -418,6 +420,18 @@ ExecCreateTableAs(CreateTableAsStmt *stmt, const char *queryString,
 								GetActiveSnapshot(), InvalidSnapshot,
 								dest, params, 0);
 
+	if (gp_enable_gpperfmon 
+			&& Gp_role == GP_ROLE_DISPATCH 
+			&& log_min_messages < DEBUG4)
```

Comment:
Why do we need to care about log level for regular CTAS?

Reply:
Fixed